<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PtzControl</class>
 <widget class="QWidget" name="PtzControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#label_title
{
	padding-top: 20px;
	padding-bottom: 10px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_background" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="label_title">
        <property name="font">
         <font>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 10px;</string>
        </property>
        <property name="text">
         <string>PTZ</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetBundleStream" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resource/resource.qrc">:/ptz/ptz/ptzstream.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ComboBox" name="comboBoxBundleStream">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="PtzControlWidget" name="widget_control" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_tab_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_light">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	outline: 0px;
	border: 0px;
	border-radius: 0px;
	color: #FFFFFF;
	min-width: 0px;
	background-color: #5F5F5F;
}
QPushButton:disabled
{
    background-color: rgb(190, 190, 190);
}
QPushButton:hover
{
	background-color: #2B9CC6;
}
QPushButton:checked
{
	background-color: #2B9CC6;
}
QPushButton:pressed
{
	background-color: rgba(43, 156, 198, 50%);
}
QPushButton:checked:disabled
{
    background-color: rgb(190, 190, 190);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resource/resource.qrc">
             <normaloff>:/ptz/ptz/light.png</normaloff>:/ptz/ptz/light.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_lensInitialize">
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	outline: 0px;
	border: 0px;
	border-radius: 0px;
	color: #FFFFFF;
	min-width: 0px;
	background-color: #5F5F5F;
}
QPushButton:disabled
{
    background-color: rgb(190, 190, 190);
}
QPushButton:hover
{
	background-color: #2B9CC6;
}
QPushButton:checked
{
	background-color: #2B9CC6;
}
QPushButton:pressed
{
	background-color: rgba(43, 156, 198, 50%);
}
QPushButton:checked:disabled
{
    background-color: rgb(190, 190, 190);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resource/resource.qrc">
             <normaloff>:/ptz/ptz/initialize.png</normaloff>:/ptz/ptz/initialize.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_autoFocus">
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	outline: 0px;
	border: 0px;
	border-radius: 0px;
	color: #FFFFFF;
	min-width: 0px;
	background-color: #5F5F5F;
}
QPushButton:disabled
{
    background-color: rgb(190, 190, 190);
}
QPushButton:hover
{
	background-color: #2B9CC6;
}
QPushButton:pressed
{
	background-color: rgba(43, 156, 198, 50%);
}
QPushButton:pressed
{
	background-color: rgba(43, 156, 198, 50%);
}
QPushButton:checked
{
	background-color: #2B9CC6;
}
QPushButton:checked:disabled
{
    background-color: rgb(190, 190, 190);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resource/resource.qrc">
             <normaloff>:/ptz/ptz/auto_focus.png</normaloff>:/ptz/ptz/auto_focus.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_tab" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_preset">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	outline: 0px;
	border: 0px;
	border-radius: 0px;
	border-top-left-radius: 5px;
	color: #FFFFFF;
	min-width: 0px;
	background-color: #5F5F5F;
}
QPushButton:disabled
{
    background-color: rgb(190, 190, 190);
}
QPushButton:hover
{
	background-color: #2B9CC6;
}
QPushButton:checked
{
	background-color: #2B9CC6;
}
QPushButton:checked:disabled
{
    background-color: rgb(190, 190, 190);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resource/resource.qrc">
             <normaloff>:/ptz/ptz/preset.png</normaloff>:/ptz/ptz/preset.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_path">
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	outline: 0px;
	border: 0px;
	border-radius: 0px;
	color: #FFFFFF;
	min-width: 0px;
	background-color: #5F5F5F;
}
QPushButton:disabled
{
    background-color: rgb(190, 190, 190);
}
QPushButton:hover
{
	background-color: #2B9CC6;
}
QPushButton:checked
{
	background-color: #2B9CC6;
}
QPushButton:checked:disabled
{
    background-color: rgb(190, 190, 190);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resource/resource.qrc">
             <normaloff>:/ptz/ptz/patrol.png</normaloff>:/ptz/ptz/patrol.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_pattern">
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	outline: 0px;
	border: 0px;
	border-radius: 0px;
	border-top-right-radius: 5px;
	color: #FFFFFF;
	min-width: 0px;
	background-color: #5F5F5F;
}
QPushButton:disabled
{
    background-color: rgb(190, 190, 190);
}
QPushButton:hover
{
	background-color: #2B9CC6;
}
QPushButton:checked
{
	background-color: #2B9CC6;
}
QPushButton:checked:disabled
{
    background-color: rgb(190, 190, 190);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resource/resource.qrc">
             <normaloff>:/ptz/ptz/pattern.png</normaloff>:/ptz/ptz/pattern.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_preset">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="PresetWidget" name="widget_preset" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_path">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="PathWidget" name="widget_path" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_pattern">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="2" column="0">
           <widget class="PatternWidget" name="widget_pattern" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="MyPushButton" name="pushButton_close">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>PtzControlWidget</class>
   <extends>QWidget</extends>
   <header>ptzcontrolwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PresetWidget</class>
   <extends>QWidget</extends>
   <header>presetwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PathWidget</class>
   <extends>QWidget</extends>
   <header>pathwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PatternWidget</class>
   <extends>QWidget</extends>
   <header>patternwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
